packages/x-tree-view/src/themeAugmentation/components.d.ts

```diff
@@ -11,6 +11,11 @@ export interface TreeViewComponents<Theme = unknown> {
     styleOverrides?: ComponentsOverrides<Theme>['MuiTreeView'];
     variants?: ComponentsVariants['MuiTreeView'];
   };
+  MuiRichTreeView?: {
```

Comment:
If we are not keeping the `TreeView`, but introducing `SimpleTreeView`, then we need to add another entry.

Reply:
Thanks, I forgot to add `SimpleTreeView` here :/